frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/LinkOptions/LinkedEntityPicker.styled.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "@emotion/styled";
+
+import { Select } from "metabase/ui";
+
+export const DashboardTabSelect = styled(Select)`
```

Comment:
mt="md"? No styled component should be needed

Reply:
5b4ec8af72d0f41896255ce307a073089e1b683b